modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc

```diff
@@ -160,6 +174,148 @@ include::{root-partials}replication-filters.adoc[]
 
 include::{root-partials}replication-channels.adoc[]
 
+
+[#anchor-auto-purge-on-revoke]
+=== Auto-purge on Channel Access Revocation
+
+.Breaking Change
+[CAUTION]
+--
+By default, when a user loses access to a channel all documents in the channel (that do not also belong to any of user’s other channels) are auto-purged from the local database (in devices belonging to the user). +
+_In release 2.x these documents will remain in the local database._
```

Comment:
I'm not sure if trying to explain it here is too complicated.

In 2.x, CBL also does auto-purge but only for the case when the user loses access to the doc via removing the doc from all of the channels belong to the user. (Not sure if we have this explained any where in 2.x doc).

In 3.0, in addition to 2.x auto purge:
*  CBL will auto-purge the docs when the user loses access to the doc via channel access revocation.
* CBL provides configuration to to enable / disable auto purge - enable by default.